assets/src/application/ui/input/index.ts

```diff
@@ -11,6 +11,6 @@ export { default as Text } from './Text';
 export { default as Hidden } from './Hidden';
 export { default as TextArea } from './TextArea';
 export { default as TimePicker } from './DateTimePicker/TimePicker';
-export { default as withLabel } from './withLabel';
```

Comment:
Since you have removed it from here, there are some consumer components like `SelectInput`, `SearchInput` etc. that will be affected.

Reply:
those have all been updated